portfolio/src/main/java/com/google/sps/servlets/GetUserDataServlet.java

```diff
@@ -9,21 +9,32 @@
 
 import com.google.appengine.api.users.UserService;
 import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.JsonObject;
 
-@WebServlet("/user")
-public class GetUserServlet extends HttpServlet {
+@WebServlet("/user-data")
+public class GetUserDataServlet extends HttpServlet {
   
   /* Validates that the user is logged in and gets their email */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     UserService userService = UserServiceFactory.getUserService();
-    response.setContentType("application/json;");
+    JsonObject json = new JsonObject();
 
     if (userService.isUserLoggedIn()) {
       String email = userService.getCurrentUser().getEmail();
-      response.getWriter().println("{\"email\": \"" + email + "\"}");
+      json.addProperty("email", email);
+
+      if (userService.isUserAdmin()) {
+        json.addProperty("admin", "true");
+      } else {
+        json.addProperty("admin", "false");
+      } 
```

Comment:
nit: this logic can be replaced by a ternary operator:

`json.addProperty("admin", userService.isUserAdmin() ? "true" : "false");`

Reply:
I tried doing
`userService.isUserAdmin() ? json.addProperty("admin", "true"): json.addProperty("admin", "false");`

But after looking it up, it seems like Java only allows ternary operators when the right side returns a value. (https://stackoverflow.com/questions/9450864/java-ternary-with-no-return-for-method-calling)

To clean it up, I instead condensed the if statement to one line.

Reply:
Oh interesting, I didn't know you could do a ternary operator in a statement!